test/src/tests-core-sdk.js

```diff
@@ -832,33 +799,28 @@ describe('core SDK', function() {
     });
 
     it('should have default urls if no custom urls are set in config object, but use custom urls when they are set', function(done) {
-        window.mParticle.config.v1SecureServiceUrl =
-            'custom-v1SecureServiceUrl/';
-        window.mParticle.config.v2SecureServiceUrl =
-            'custom-v2SecureServiceUrl/v2/JS/';
         window.mParticle.config.v3SecureServiceUrl =
-            'custom-v3SecureServiceUrl/v3/JS/';
+            'custom-v3secureserviceurl/v3/JS/';
         window.mParticle.config.configUrl =
             'custom-configUrl/v2/JS/';
         window.mParticle.config.identityUrl = 'custom-identityUrl/';
         window.mParticle.config.aliasUrl = 'custom-aliasUrl/';
 
         mParticle.init(apiKey, window.mParticle.config);
 
-        mParticle.getInstance()._Store.SDKConfig.v1SecureServiceUrl.should.equal(window.mParticle.config.v1SecureServiceUrl)
-        mParticle.getInstance()._Store.SDKConfig.v2SecureServiceUrl.should.equal(window.mParticle.config.v2SecureServiceUrl)
         mParticle.getInstance()._Store.SDKConfig.v3SecureServiceUrl.should.equal(window.mParticle.config.v3SecureServiceUrl)
         mParticle.getInstance()._Store.SDKConfig.configUrl.should.equal(window.mParticle.config.configUrl)
         mParticle.getInstance()._Store.SDKConfig.identityUrl.should.equal(window.mParticle.config.identityUrl)
         mParticle.getInstance()._Store.SDKConfig.aliasUrl.should.equal(window.mParticle.config.aliasUrl)
 
         // test events endpoint
-        mockServer.requests = [];
-        mParticle.logEvent('test');
-        mockServer.requests[0].url.should.equal(
+        mParticle.logEvent('Test Event');
+
+        var testEventURL = findRequestURL(window.fetchMock._calls, 'Test Event');
+        testEventURL.should.equal(
             'https://' +
-                window.mParticle.config.v2SecureServiceUrl +
-                'test_key/Events'
+                window.mParticle.config.v3SecureServiceUrl +
```

Comment:
technically there's a v1 still because the forwarding address uses the v1 endpoint but with /Forwarding appended.  so to keep those separate i'd say no for now.  there's probably a more thoughtful way to do this once we migrate to TS and refactor to make it a class with property as opposed to everything being on the config though.